great_expectations/data_context/store/expectations_store.py

```diff
@@ -111,6 +114,38 @@ def gx_cloud_response_json_to_object_dict(response_json: Dict) -> Dict:
 
         return suite_dict
 
+    def add_expectation(
+        self, suite: ExpectationSuite, expectation: Expectation
+    ) -> Expectation:
+        suite_identifier = self.get_key(suite=suite)
+        suite_dict = self.get(key=suite_identifier)
+        suite = ExpectationSuite(**suite_dict)
+
+        expectation.id = str(uuid.uuid4())
+        suite.expectation_configurations.append(expectation.configuration)
+
+        self.update(key=suite_identifier, value=suite)
+        return expectation
+
+    def update_expectation(
+        self, suite: ExpectationSuite, expectation: Expectation
+    ) -> Expectation:
+        suite_identifier = self.get_key(suite=suite)
+        suite_dict = self.get(key=suite_identifier)
+        suite = ExpectationSuite(**suite_dict)
```

Comment:
Could you put this in a method called something like `refresh_suite`? We do this for the other crud operations also and encapsulating into something with a name would make this more readable for me. I'm surprised `delete` isn't here?

Reply:
thanks - `delete` is required, i'll add it

Reply:
`delete_expectation` is added in [60a7207](https://github.com/great-expectations/great_expectations/pull/9125/commits/60a72071bca34e62eed2fc7365a64f571fda88e7), `_refresh_suite` added in [c71e322](https://github.com/great-expectations/great_expectations/pull/9125/commits/c71e322ef4eb0919817517a7da17239d5adb1608)